internal/services/v1/relationships.go

```diff
@@ -73,6 +73,22 @@ type PermissionsServerConfig struct {
 	// MaxCheckBulkConcurrency defines the maximum number of concurrent checks that can be
 	// made in a single CheckBulkPermissions call.
 	MaxCheckBulkConcurrency uint16
+
+	// MaxReadRelationshipsLimit defines the maximum number of relationships that can be read
+	// in a single ReadRelationships call.
+	MaxReadRelationshipsLimit uint32
+
+	// MaxDeleteRelationshipsLimit defines the maximum number of relationships that can be deleted
+	// in a single DeleteRelationships call.
+	MaxDeleteRelationshipsLimit uint32
+
+	// MaxLookupResourcesLimit defines the maximum number of resources that can be looked up in a
+	// single LookupResources call.
+	MaxLookupResourcesLimit uint32
+
+	// MaxBulkExportRelationshipsLimit defines the maximum number of relationships that can be
+	// exported in a single BulkExportRelationships call.
+	MaxBatchExportRelationshipsLimit uint32
```

Comment:
should use the terminology of the API
```suggestion
	MaxBulkExportRelationshipsLimit uint32
```

Reply:
whoops.